### PR TITLE
Sneha, Vinay, Alekhya | MOBN-984 | Add focus lock for success and con…

### PR DESCRIPTION
**Pull request by AlekhyaYalla**
…flicts pop up

### CODE REVIEW

File: ui/react-components/components/Conflicts/Conflicts.jsx [hunk 1]
```diff
@@ -5,14 +5,15 @@ import PropTypes from "prop-types";
 import {injectIntl} from "react-intl";
 import classNames from 'classnames';
 import {button} from "./Conflicts.module.scss";
+import WithFocusLock from "../FocusLock/WithFocusLock.jsx";
```
Comment (bsneha90) [inline ui/react-components/components/Conflicts/Conflicts.jsx]: naming convention is useFocusLock, please change

File: ui/react-components/components/FocusLock/WithFocusLock.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import FocusLock from "react-focus-lock";
```
Comment (bsneha90) [inline ui/react-components/components/FocusLock/WithFocusLock.jsx]: Can you move this to hooks folder under util.

File: ui/react-components/components/Conflicts/Conflicts.jsx (hunk 1 above)
Comment (AlekhyaYalla) [reply ui/react-components/components/Conflicts/Conflicts.jsx]: Done

File: ui/react-components/components/FocusLock/WithFocusLock.jsx (hunk 2 above)
Comment (AlekhyaYalla) [reply ui/react-components/components/FocusLock/WithFocusLock.jsx]: Done

**Pull request merged**